akkumulator/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/akkumulator/Akkumulator.kt

```diff
@@ -2,89 +2,113 @@
 
 package no.nav.helsearbeidsgiver.inntektsmelding.akkumulator
 
+import com.fasterxml.jackson.databind.JsonNode
 import com.fasterxml.jackson.databind.ObjectMapper
 import com.fasterxml.jackson.databind.node.ObjectNode
 import no.nav.helse.rapids_rivers.JsonMessage
 import no.nav.helse.rapids_rivers.MessageContext
-import no.nav.helse.rapids_rivers.MessageProblems
 import no.nav.helse.rapids_rivers.RapidsConnection
 import no.nav.helse.rapids_rivers.River
 import no.nav.helse.rapids_rivers.isMissingOrNull
+import no.nav.helsearbeidsgiver.felles.Key
+import no.nav.helsearbeidsgiver.felles.value
 
 class Akkumulator(
     rapidsConnection: RapidsConnection,
     private val redisStore: RedisStore,
     private val timeout: Long = 600
 ) : River.PacketListener {
 
-    val objectMapper = ObjectMapper()
+    private val objectMapper = ObjectMapper()
 
     init {
         River(rapidsConnection).apply {
             validate {
-                it.requireKey("@id")
-                it.requireKey("uuid")
```

Comment:
Hvorfor fjerner du at disse er required? Vi må ha noe som er unikt for hver hendelse